service/src/main/java/bio/terra/tanagra/app/authentication/AuthInterceptor.java

```diff
@@ -82,20 +93,28 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
                 jwt,
                 authenticationConfiguration.getGcpProjectNumber(),
                 authenticationConfiguration.getGkeBackendServiceId());
+
```

Comment:
Is this a common style in VWB? I haven't seen it before.

Reply:
only added to improve readability, new line not enforced here. We use spotlessJava to enforce general code style.